pkgs/development/python-modules/psycopg2cffi/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, cffi, postgresql, openssl, six }:
+
+buildPythonPackage rec {
+  pname = "psycopg2cffi";
+  version = "2.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yc0cxxkfr35kd959wsagxfhy6ikbix71rp9x0rmn858fxa4vapw";
+  };
+
+  buildInputs = [ six ] ++ lib.optional stdenv.isDarwin openssl;
+  nativeBuildInputs = [ cffi postgresql ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Please include a comment.

Reply:
The same reason they're disabled here (with no comment) https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/psycopg2/default.nix#L17 since this is a fork - you'd presumably know the reason better than me...